### PR TITLE
Initial setup

### PR DESCRIPTION
**Pull request by sagesmith-wf**
#### Changes:
- Initial setup

---
#### Please Review:

@sagersmith8 @Kuroshii @bradbenjamin-wf

### CODE REVIEW

File: .flake8 [hunk 1]
```diff
@@ -0,0 +1,4 @@
+[flake8]
+ignore = E226,E302,E41
+max-line-length = 80
+max-complexity = 10
```
Comment (sagesmith-wf) [inline .flake8]: New line needed

File: app.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  script: /src/main.py
+
+builtins:
+- appstats: on
+
+libraries:
+- name: webapp2
+  version: "2.5.2"
+- name: jinja2
+  version: "latest"
```
Comment (sagesmith-wf) [inline app.yaml]: Needs newline

File: .flake8 [hunk 3]
```diff
@@ -0,0 +1,4 @@
+[flake8]
+ignore = E226,E302,E41
+max-line-length = 80
```
Comment (bradbenjamin-wf) [inline .flake8]: FLAME WAR!

File: run-tests.sh [hunk 4]
```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+set -ex
```
Comment (Kuroshii) [inline run-tests.sh:2]: +1

**Pull request merged**